client/my-sites/checkout/composite-checkout/wpcom/components/wp-order-review-line-items.js

```diff
@@ -32,14 +32,15 @@ function WPLineItem( {
 	getItemVariants,
 	onChangePlanLength,
 	isSummary,
+	isPwpoUser,
 } ) {
 	const translate = useTranslate();
 	const hasDomainsInCart = useHasDomainsInCart();
 	const { formStatus } = useFormStatus();
 	const itemSpanId = `checkout-line-item-${ item.id }`;
 	const deleteButtonId = `checkout-delete-button-${ item.id }`;
 	const [ isModalVisible, setIsModalVisible ] = useState( false );
-	const modalCopy = returnModalCopy( item.type, translate, hasDomainsInCart );
+	const modalCopy = returnModalCopy( item.type, translate, hasDomainsInCart, isPwpoUser );
```

Comment:
This implementation is fine, but rather than passing this prop down from `CompositeCheckout`, we could just call `currentUserHasFlag` here instead. What do you think?

Reply:
I've pushed this change